code/modules/holiday/holidays.dm

```diff
@@ -693,7 +693,7 @@
 
 /datum/holiday/xmas
 	name = CHRISTMAS
-	begin_day = 23
+	begin_day = 8
```

Comment:
either add monkestation edit, or just override in like `monkestation/code/modules/holiday/holidays.dm`